lib/Config.js

```diff
@@ -32,9 +32,10 @@ exports.settings = {
   no_answer_timeout              : 60,
 
   // Registration parameters.
-  register         : true,
-  register_expires : 600,
-  registrar_server : null,
+  register                : true,
+  register_expires        : 600,
+  register_from_tag_trail : '',
```

Comment:
Everything related to `register_from_tag_trail` does not belong to this PR. It's already in master. Please remove it.

Reply:
Done